retain-spec/openapi.json

```diff
@@ -641,6 +665,14 @@
             ],
             "description": "Indicates whether client has consent for SMS communications with this customer, at this number. If the telephoneSMSConsent is being updated, the associated phone number must also be provided."
           },
+          "telephoneVMConsent1": {
+            "type": "string",
+            "enum": [
+              "YES",
+              "NO"
+            ],
+            "description": "Indicates whether client has consent for Voicemail Drop communications with this customer, at this number."
```

Comment:
Do we want to add analogous text from the SMS consent field above? `If the telephoneSMSConsent is being updated, the associated phone number must also be provided`

Reply:
✅